incubator/hnc/pkg/controllers/config_controller_test.go

```diff
@@ -52,6 +54,29 @@ var _ = Describe("HNCConfiguration", func() {
 		Eventually(hasSecret(ctx, barName, "foo-sec")).Should(BeTrue())
 		Expect(secretInheritedFrom(ctx, barName, "foo-sec")).Should(Equal(fooName))
 	})
+
+	It("should propagate an empty object whose types have been added to HNCConfiguration", func() {
```

Comment:
Ideally this would be in object-controller-test, since it's only testing a pure feature of the object controller? Would it be possible to move this there?

Reply:
Good idea. Moved to the object_controller_test.go. Also moved some more shared functions to the test_helpers_test.go